tests/wpt/metadata/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/fieldset-transform-translatez.html.ini

```diff
@@ -0,0 +1,2 @@
+[fieldset-transform-translatez.html]
+  expected: FAIL
```

Comment:
Hmm, a new `FAIL`?

Reply:
There are minute pixel differences in the text rendering :/
<img width="810" alt="Screen Shot 2019-11-27 at 1 47 28 PM" src="https://user-images.githubusercontent.com/27658/69751147-781acc00-111c-11ea-8233-b4e0117e828b.png">

Reply:
Oh that's annoying :/